app/models/tag.rb

```diff
@@ -89,6 +91,10 @@ def update_parent(representative)
     update(representative: representative)
   end
 
+  def representative
+    super || self
+  end
+
```

Comment:
Hmm... could there be an alternative to this? I think some queries in some other sections of the app fetch reps by looking for tags with `nil` values in their rep column.